app/models/apple/publisher.rb

```diff
@@ -392,5 +413,51 @@ def remove_audio_container_reference(eps, apple_mark_for_reupload: true)
         Apple::Episode.remove_audio_container_reference(api, show, eps, apple_mark_for_reupload: apple_mark_for_reupload)
       end
     end
+
+    private
+
+    def check_for_stuck_episodes(waiting)
```

Comment:
And this gets invoked every time we call the publishing process, yeah?  Which, will keep happening for a Podcast as long as there are episodes to deliver to Apple, I think.